using-timescaledb/move_cunks.md

```diff
@@ -0,0 +1,187 @@
+of storage it resides on by simply adding new tablespaces backed by different 
+classes of storage.
+
+Let’s take a look at an example:
+
+![alt text](https://assets.timescale.com/images/diagrams/move_chunks_diagram.png "Move_chunks")
+
+In the above example, data moves over time from left (position 0) to right (position 4). 
+We can use additional tablespaces in Postgres backed by slower and less expensive 
+storage as data ages. The real benefit here is the ability to manage this data in 
+TimescaleDB at the chunk level. 
+
+As chunks age you can move them from left to right (as seen above) to reduce the 
+costs associated with storing data you need to keep. Combined with compression, 
+we enable users to tightly manage costs without needing to purge data (an option 
+that can be difficult or impossible to use in a compliance driven environment, or 
+where historical data analysis is a key part of your work).
+
+Using multiple tablespaces also yields I/O performance benefits. With data tiering 
+you gain the ability to spread the read I/O load across multiple storage mounts and 
+disk arrays, and ensure that large hypertables (where you may be performing analytics) 
+do not overrun the disk backing the default tablespace.
+
+
+### Move Chunks :enterprise_function:[] (move) 
+The move chunks function requires multiple tablespaces set up in Postgres, so let's 
+start with a quick review of how this works.
+
+First, add a storage mount that will serve as a home for your new tablespace. This 
+process will differ based on how you are deployed, but your system administrator 
+should be able to arrange setting up the mount point. The key here is to backend 
+your tablespace with storage that is appropriate for how its resident data will be used.
+
+To create a [tablespace] in Postgres:
+
+```sql
+CREATE TABLESPACE tablespace_2
+OWNER postgres
+LOCATION '/mnt/postgres';
+```
+
+Here we are creating a tablespace called tablespace_2 that will be in this case owned 
+by the Postgres user and it will use the storage mounted at ```/mnt/postgres```.
+
+Now we are ready to leverage this new tablespace and underlying storage for TimescaleDB 
+chunks and their indexes. 
+
+---
+
+### Chunks and Indexes on the same tablespace [](elements)
+
+Now that we have set up a new, empty tablespace,we can move chunks (along with their 
+indexes) off of their default tablespace. 
+
+To determine which chunks to move, we can list chunks that fit a specific criteria. 
+For example,  to identify chunks older than two days:
+
+```sql
+SELECT show_chunks('conditions', older_than => interval '2 days');
+```
+
+In this example, we will move 1_4 along with its index over to tablespace_2 using the following command:
+
+```sql
+SELECT move_chunk(chunk=>'_timescaledb_internal._hyper_1_4_chunk', destination_tablespace=>'tablespace_2', 
+index_destination_tablespace=>'tablespace_2', reorder_index=>'_timescaledb_internal._hyper_1_4_chunk_netdata_time_idx', 
```

Comment:
we never cover the reorder_index required option, and I think we should.

Reply:
I agree but I will need some help with that can we cover that post release?